e2e-tests/ukef/cypress/e2e/journeys/utilisation-reports/utilisation-report-submission.spec.js

```diff
@@ -0,0 +1,235 @@
+    const notReceivedFebruary2023Report = UtilisationReportEntityMockBuilder.forStatus(UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED)
+      .withId(BANK1_PAYMENT_REPORT_OFFICER1.bank.id + 10000000)
+      .withBankId(BANK1_PAYMENT_REPORT_OFFICER1.bank.id)
+      .withReportPeriod({ start: { month: 2, year: 2023 }, end: { month: 2, year: 2023 } })
+      .build();
+    cy.task(NODE_TASKS.INSERT_UTILISATION_REPORTS_INTO_DB, [notReceivedFebruary2023Report]);
+
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+  });
+
+  after(() => {
+    cy.clearCookies();
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+
+    cy.task(NODE_TASKS.DELETE_ALL_FROM_SQL_DB);
+    cy.task(NODE_TASKS.DELETE_ALL_TFM_FACILITIES_FROM_DB);
+  });
+
+  /**
+   * Asserts the values in a given row of the premium payments table match the expected
+   * values from the report as defined in february2023ExpectedValues
+   * @param {number} premiumPaymentsRowIndex - The index of the row of the premium payments table
+   * @param {object} expectedParsedValuesForReportRow - The expected values to be found in the row
+   */
+  const assertPremiumPaymentsTableRowContainsExpectedValues = (premiumPaymentsRowIndex, expectedParsedValuesForReportRow) => {
+    cy.assertText(
+      tfmPages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.facilityIdByRowIndex(premiumPaymentsRowIndex),
+      expectedParsedValuesForReportRow.facilityId,
+    );
+    cy.assertText(
+      tfmPages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.exporterIdByRowIndex(premiumPaymentsRowIndex),
+      expectedParsedValuesForReportRow.exporter,
+    );
+    cy.assertText(
+      tfmPages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.reportedFeesByRowIndex(premiumPaymentsRowIndex),
+      expectedParsedValuesForReportRow.reportedFees,
+    );
+    cy.assertText(
+      tfmPages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.reportedPaymentsByRowIndex(premiumPaymentsRowIndex),
+      expectedParsedValuesForReportRow.reportedPayments,
+    );
+    cy.assertText(
+      tfmPages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.totalReportedPaymentsByRowIndex(premiumPaymentsRowIndex),
+      expectedParsedValuesForReportRow.reportedPayments,
+    );
+  };
+
+  /**
+   * Asserts the values in a given row of the utilisation table match the expected
+   * values from the report as defined in february2023ExpectedValues
+   * @param {number} utilisationTableRowIndex - The index of the row of utilisation table
+   * @param {object} expectedParsedValuesForReportRow - The expected values to be found in the row
+   */
+  const assertUtilisationTableRowContainsExpectedValues = (utilisationTableRowIndex, expectedParsedValuesForReportRow) => {
```

Comment:
Can we rename `expectedParsedValuesForReportRow` to something simpler like `expectedValues`?

Reply:
bump @RLavender98